### PR TITLE
Btn color consistency

### LINKED ISSUE #43: [Bug]: Button colors are different
### What happened?

On the home page, we have a normal background color for buttons and on the other page, we have a gradient background color, we should have the gradient color for all the buttons.

Steps

- Check out this line in the fallback component https://github.com/OpenSource-Journey/Your-Github-Contributions/blob/main/src/components/ErrorFallback/FallbackComponent.tsx#L24
- Make the changes in this file https://github.com/OpenSource-Journey/Your-Github-Contributions/blob/main/src/components/Forms/UserNamForm.tsx#L54

### Please provide an bug screenshot

- Error page
image

- Home Page
image

### What browsers are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mayukh551**
## Pull Request Checklist

@Sachin-chaurasiya 

Before submitting a pull request, please make sure that your changes meet the following criteria:

- [ ] Ensure that your code adheres to our coding conventions and is properly formatted.
- [ ] Test your changes extensively to ensure that they do not introduce any new bugs.
- [ ] Confirm that you agree to the code of conduct.

## Related Issues

If this pull request is related to any issues, please list them here.
#43 

## Screenshots

If your changes include any visual elements, please provide screenshots or other visual aids to help reviewers understand your changes.

## Additional Notes

If there is anything else you would like to note about your changes or the review process, please add it here.

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@mayukh551 , Thanks for the PR, a couple of things.
- Do not push the package-lock.json file ( for the development purpose we recommend using the `yarn`)
- Do not upgrade or uninstall the packages ( if not needed )
- Your PR should be scoped to the related issue.

File: README.md [hunk 1]
```diff
@@ -62,7 +62,6 @@ Your GitHub Contributions was built using the following technologies:
 - [ReactJs](https://reactjs.org/) - A JavaScript library for building user interfaces
 - [ChakraUI](https://chakra-ui.com/) - A simple, modular and accessible component library
 
-
```
Comment (Sachin-chaurasiya) [inline README.md:65]: @mayukh551 , Can you remove this change?

File: package-lock.json
Comment (Sachin-chaurasiya) [inline package-lock.json:1]: @mayukh551 , Please do not push this file.

File: package.json
Comment (Sachin-chaurasiya) [inline package.json:1]: @mayukh551 , Please revert these file changes.

File: yarn.lock
Comment (Sachin-chaurasiya) [inline yarn.lock:1]: @mayukh551 , Please revert these file changes.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Left a few comments, thanks

**Comment by mayukh551** [comment]
@Sachin-chaurasiya I understand where I made the mistake. Since, it's a small change, I am going to create a new PR by closing this one.

**Pull request closed without merging**